cmd/dagger/flags.go

```diff
@@ -318,7 +318,7 @@ func (v *directoryValue) Get(ctx context.Context, dag *dagger.Client, modSrc *da
 	gitURL, err := parseGitURL(v.String())
 	if err == nil {
 		// TODO: use modArg.Ignore if not empty
```

Comment:
This TODO can be removed. You can also remove the other one further down on `dag.Host` because we don't have any core functions with `Ignore`. Doesn't hurt to support but I also don't see the point, at least not prematurely.

Reply:
True! Okay I'll fix that